scripts/e2e-test-op-commands.py

```diff
@@ -0,0 +1,137 @@
+import json
+import os
+from pathlib import Path
+from typing import Any
+
+import click
+import structlog
+from eth_account.account import LocalAccount
+from eth_utils import to_checksum_address
+from hexbytes import HexBytes
+from web3 import Web3
+
+from beamer.contracts import contracts_for_web3, make_contracts, prepare_deployment_infos
+from beamer.typing import URL, ChainId
+from beamer.util import account_from_keyfile, make_web3, transact
+from scripts.call_contracts import pass_args
+
+logger = structlog.get_logger(__name__)
+
+FAILED_RELAYED_MESSAGE_TOPIC = HexBytes(
+    "0x99d0e048484baa1b1540b1367cb128acd7ab2946d1ed91ec10e3c85e4bf51b8f"
+)
+WITHDRAWAL_FINALIZED_TOPIC = HexBytes(
+    "0xdb5c7652857aa163daadd670e116628fb42e869d8ac4251ef8971d9e5727df1b"
+)
+RELAYED_MESSAGE_TOPIC = HexBytes(
+    "0x4641df4a962071e12719d8c8c8e5ac7fc4d97b927346a3d7a335b1f7517e133c"
+)
+
+
+@click.group("e2e-op-commands")
+@click.argument(
+    "keystore-file",
+    type=click.Path(exists=True, dir_okay=False, readable=True),
+    required=True,
+)
+@click.argument("password", type=str, required=True)
+@click.argument("l1-rpc", type=str, required=True)
+@click.pass_context
+def cli(ctx: Any, keystore_file: Path, password: str, l1_rpc: str) -> None:
+    account = account_from_keyfile(keystore_file, password)
+    web3_l1 = make_web3(URL(l1_rpc), account)
+    ctx.ensure_object(dict)
+    ctx.obj["account"] = account
+    ctx.obj["web3_l1"] = web3_l1
+
+
+@cli.command("verify-portal-call")
+@click.argument(
+    "portal-address",
+    type=str,
+)
+@click.argument(
+    "messenger-address",
+    type=str,
+)
+@pass_args
+def verify_portal_call(
+    account: LocalAccount,  # pylint:disable=unused-argument
+    web3_l1: Web3,
+    portal_address: str,
+    messenger_address: str,
+) -> None:
+    portal_logs = web3_l1.eth.get_logs(
+        {"address": to_checksum_address(portal_address), "fromBlock": 0}
+    )
+    messenger_logs = web3_l1.eth.get_logs(
+        {"address": to_checksum_address(messenger_address), "fromBlock": 0}
+    )
+    assert any(WITHDRAWAL_FINALIZED_TOPIC in log["topics"] for log in portal_logs)
+    assert any(FAILED_RELAYED_MESSAGE_TOPIC in log["topics"] for log in messenger_logs)
+    print("Processed Block Number:", web3_l1.eth.block_number)
+
+
+@cli.command("set-chain-on-resolver")
+@click.argument(
+    "deployment-dir",
+    type=Path,
+    required=True,
+)
+@click.argument("l2-rpc", type=str)
+@pass_args
+def set_messenger_on_resolver(
+    account: LocalAccount, web3_l1: Web3, deployment_dir: Path, l2_rpc: URL
+) -> None:
+    web3 = make_web3(l2_rpc, account)
+    env_chain_id = os.getenv("SOURCE_CHAIN_ID")
+    assert env_chain_id is not None
+    target_chain_id = ChainId(int(env_chain_id))
```

Comment:
You can replace these with just one line:
```python
    target_chain_id = ChainId(int(os.environ['SOURCE_CHAIN_ID']))
```
If the env variable is not there, you will get an except, same as on `assert`.

Also, why are we calling it `target_chain_id` when we are using `SOURCE_CHAIN_ID` for the env variable?